### ISSUE TITLE
put example on gh-pages branch

### ISSUE THREAD

**Issue by zackify** [dphaener/reactable-popover#3]
yeah? :P

**Comment by dphaener** [comment]
Yeah, I've never made a gh-pages branch before. Is there any secret magic?

**Comment by zackify** [comment]
This is why we need a small script in the generator to do this:
- git checkout -b gh-pages
- rm -rf everything except /example
- mv example .
- git add .
- git commit -m 'updated example'
- git push
